### ISSUE TITLE
Crash on server start 1.21.1

### ISSUE THREAD

**Issue by 27rogi** [DivineRPG/DivineRPG#696]
I'm unable to start server with DivineRPG on latest NeoForge 1.21.1. Not sure if this error is intended. Singleplayer works fine.

```java
-- Mod loading issue for: divinerpg --
Details:
	Caused by 0: java.lang.BootstrapMethodError: java.lang.RuntimeException: Attempted to load class divinerpg/events/ArcanaRenderer for invalid dist DEDICATED_SERVER
		at TRANSFORMER/divinerpg@1.10.8/divinerpg.DivineRPG.<init>(DivineRPG.java:56) ~[divinerpg-1.10.8.jar%23270!/:?] {re:classloading}
		at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62) ~[?:?] {}
		at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502) ~[?:?] {}
		at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486) ~[?:?] {}
		at MC-BOOTSTRAP/fml_loader@4.0.34/net.neoforged.fml.javafmlmod.FMLModContainer.constructMod(FMLModContainer.java:115) ~[loader-4.0.34.jar%2360!/:4.0] {}
		at MC-BOOTSTRAP/fml_loader@4.0.34/net.neoforged.fml.ModLoader.lambda$constructMods$3(ModLoader.java:153) ~[loader-4.0.34.jar%2360!/:4.0] {}
		at MC-BOOTSTRAP/fml_loader@4.0.34/net.neoforged.fml.ModLoader.lambda$dispatchParallelTask$7(ModLoader.java:231) ~[loader-4.0.34.jar%2360!/:4.0] {}
		at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934) ~[?:?] {re:mixin}
		at java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911) ~[?:?] {}
		at java.base/java.util.concurrent.CompletableFuture$Completion.exec(CompletableFuture.java:483) ~[?:?] {}
		at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:387) ~[?:?] {}
		at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1312) ~[?:?] {}
		at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1843) ~[?:?] {re:mixin,re:computing_frames}
		at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1808) ~[?:?] {re:mixin,re:computing_frames}
		at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:188) ~[?:?] {re:mixin}

	Mod file: /data/mods/divinerpg-1.10.8.jar
	Failure message: DivineRPG (divinerpg) has failed to load correctly
		java.lang.BootstrapMethodError: java.lang.RuntimeException: Attempted to load class divinerpg/events/ArcanaRenderer for invalid dist DEDICATED_SERVER
	Mod version: 1.10.8
	Mod issues URL: https://github.com/DivineRPG/DivineRPG/issues
	Exception message: java.lang.RuntimeException: Attempted to load class divinerpg/events/ArcanaRenderer for invalid dist DEDICATED_SERVER
Stacktrace:
	at MC-BOOTSTRAP/fml_loader@4.0.34/net.neoforged.fml.common.asm.RuntimeDistCleaner.processClassWithFlags(RuntimeDistCleaner.java:60) ~[loader-4.0.34.jar%2360!/:4.0] {}
	at MC-BOOTSTRAP/cpw.mods.modlauncher@11.0.4/cpw.mods.modlauncher.LaunchPluginHandler.offerClassNodeToPlugins(LaunchPluginHandler.java:94) ~[modlauncher-11.0.4.jar%2362!/:?] {}
	at MC-BOOTSTRAP/cpw.mods.modlauncher@11.0.4/cpw.mods.modlauncher.ClassTransformer.transform(ClassTransformer.java:120) ~[modlauncher-11.0.4.jar%2362!/:?] {}
	at MC-BOOTSTRAP/cpw.mods.modlauncher@11.0.4/cpw.mods.modlauncher.TransformingClassLoader.maybeTransformClassBytes(TransformingClassLoader.java:57) ~[modlauncher-11.0.4.jar%2362!/:?] {}
	at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.readerToClass(ModuleClassLoader.java:190) ~[securejarhandler-3.0.8.jar:?] {}
	at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.lambda$findClass$20(ModuleClassLoader.java:298) ~[securejarhandler-3.0.8.jar:?] {}
	at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.loadFromModule(ModuleClassLoader.java:311) ~[securejarhandler-3.0.8.jar:?] {}
	at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.findClass(ModuleClassLoader.java:298) ~[securejarhandler-3.0.8.jar:?] {}
	at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:214) ~[securejarhandler-3.0.8.jar:?] {}
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526) ~[?:?] {}
	at TRANSFORMER/divinerpg@1.10.8/divinerpg.DivineRPG.<init>(DivineRPG.java:56) ~[divinerpg-1.10.8.jar%23270!/:?] {re:classloading}
	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62) ~[?:?] {}
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502) ~[?:?] {}
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486) ~[?:?] {}
	at MC-BOOTSTRAP/fml_loader@4.0.34/net.neoforged.fml.javafmlmod.FMLModContainer.constructMod(FMLModContainer.java:115) ~[loader-4.0.34.jar%2360!/:4.0] {}
	at MC-BOOTSTRAP/fml_loader@4.0.34/net.neoforged.fml.ModLoader.lambda$constructMods$3(ModLoader.java:153) ~[loader-4.0.34.jar%2360!/:4.0] {}
	at MC-BOOTSTRAP/fml_loader@4.0.34/net.neoforged.fml.ModLoader.lambda$dispatchParallelTask$7(ModLoader.java:231) ~[loader-4.0.34.jar%2360!/:4.0] {}
	at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934) ~[?:?] {re:mixin}
	at java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911) ~[?:?] {}
	at java.base/java.util.concurrent.CompletableFuture$Completion.exec(CompletableFuture.java:483) ~[?:?] {}
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:387) ~[?:?] {}
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1312) ~[?:?] {}
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1843) ~[?:?] {re:mixin,re:computing_frames}
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1808) ~[?:?] {re:mixin,re:computing_frames}
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:188) ~[?:?] {re:mixin}


```

**Comment by velocityraptor-real** [comment]
fixed in the new hotfix update 1.10.8.1

**Comment by 27rogi** [comment]
Great to hear. Thanks, will try it out soon.
